src/main/java/oshi/software/os/windows/WindowsFileSystem.java

```diff
@@ -52,53 +49,44 @@ public WindowsFileSystem() {
 
     /**
      * Gets File System Information.
-     * 
+     *
      * @return An array of {@link OSFileStore} objects representing mounted
      *         volumes. May return disconnected volumes with
      *         {@link OSFileStore#getTotalSpace()} = 0.
      */
     public OSFileStore[] getFileStores() {
-        // File.listRoots() has more information for Windows
-        // than FileSystem.getDefalut().getFileStores()
-        final File[] roots = File.listRoots();
-        // Need to call FileSystemView on Swing's Event Dispatch Thread to avoid
-        // problems
-        SwingWorker<List<OSFileStore>, Void> worker = new SwingWorker<List<OSFileStore>, Void>() {
-            @Override
-            public List<OSFileStore> doInBackground() {
-                FileSystemView fsv = FileSystemView.getFileSystemView();
-                List<OSFileStore> fsList = new ArrayList<>();
-                for (File f : roots) {
-                    String type = "unknown";
-                    String path = "unknown";
-                    try {
-                        // add trailing slash to path if needed
-                        path = f.getCanonicalPath();
-                        if (path.charAt(path.length() - 1) != '\\') {
-                            path = path + '\\';
-                        }
-                        char[] fstype = new char[16];
-                        if (Kernel32.INSTANCE.GetVolumeInformation(path, null, 0, null, null, null, fstype, 16)) {
-                            type = new String(fstype).trim();
-                        }
-                    } catch (IOException e) {
-                        LOG.error("Could not get canonical path for {}", f.toString());
-                    }
-                    fsList.add(new OSFileStore(fsv.getSystemDisplayName(f), path, fsv.getSystemTypeDescription(f), type,
-                            f.getUsableSpace(), f.getTotalSpace()));
-                }
-                return fsList;
-            }
-        };
-        worker.execute();
         List<OSFileStore> fs = new ArrayList<>();
-        try {
-            // TODO: Consider a timeout version of this method that passes
-            // timeout parameters which are used in this get()
-            fs = worker.get();
-        } catch (InterruptedException | ExecutionException e) {
-            LOG.error("", e);
+        WinDef.DWORD LENGTH = new WinDef.DWORD(255L);

```

Comment:
You go to all the trouble to define this as a `DWORD` but then only ever use its `intValue()`.   Either define it as an `int`, or if you need it as a `DWORD` (Possibly to pass to `GetVolumeInformation()` and friends) actually use the variable rather than the number 255 in those function calls.  

Also as all caps this implies it's a constant.  Define it as static final outside this method or, possibly, find the equivalent constant already existing in `Kernel32` class.  And it should be more descriptive, like VOLUME_NAME_SIZE or similar.

May need a similar constant for the 16 being thrown around here as well.